### PR TITLE
Create sample.dagql

### PR DESCRIPTION
**Pull request by CoffeeTableEspresso**
a few samples

### CODE REVIEW

**Comment by michaelenglo** [comment]
Is this ready to be merged?

**Review by michaelenglo — APPROVED** [review]
We can always modify this later, I would just merge it now

**Comment by enjmiah** [comment]
I don't see the need to merge it now?  I mean, nothing depends on it.

File: sample.dagql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+SELECT a FROM EDGES WHERE TRUE
+SELECT a FROM EDGES WHERE FALSE
+SELECT * FROM EDGES WHERE type = "process_memory"
+SELECT * FROM EDGES WHERE NOT type = "process_memory"
+SELECT * FROM EDGES WHERE type = "process_memory" AND id = 24102
+SELECT * FROM EDGES WHERE type = "process_memory" OR type = "task"
+
+-- Query specifying how to traverse
+SELECT a, b FROM NODES TRAVERSING BY DEPTH 
+SELECT a, b FROM NODES TRAVERSING BY TOPOLOGICAL
```
Comment (enjmiah) [inline sample.dagql:19]: We use `TRAVERSE` instead of `TRAVERSING` and `BREADTH` instead of `TOPOLOGICAL` now.

**Comment by michaelenglo** [comment]
I think we can just close this PR since I have already put all the sample programs as test cases in `test_parser.py`.

**Pull request closed without merging**